docs/sources/next/results-output/web-dashboard/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Web dashboard
+description: Track test results in real-time with the web-dashboard and generate HTML test reports directly from your web browser.
+weight: 200
+---
+
+# Web dashboard
+
+k6 provides a built-in web dashboard that you can enable to visualize and monitor your tests results in real-time.
+
+![Web dashboard screenshot](/media/docs/k6-oss/web-dashboard-overview.png)
+
+The dashboard provides a real-time overview of the performance observed by k6 while a test is running, and can help you identify potential reliability issues as they occur.
+
+## Generate HTML test reports
+
+You can generate detailed, downloadable HTML reports directly from the dashboard. These reports are self-contained, making them ideal for sharing with your team.
+
+![HTML test report screenshot](/media/docs/k6-oss/web-dashboard-report.png)
+
+You can access the HTML test report by clicking the "Report" button on the dashboard's menu.
+
+![HTML test report generation button](/media/docs/k6-oss/web-dashboard-report-button.png)
```

Comment:
````suggestion
You can generate detailed, downloadable HTML reports directly from the web dashboard or the command line. These reports are self-contained, making them ideal for sharing with your team.

![HTML test report screenshot](/media/docs/k6-oss/web-dashboard-report.png)

### Generate report from web dashboard

To generate a report from the web dashboard, click **Report** on the dashboard's menu.

![HTML test report generation button](/media/docs/k6-oss/web-dashboard-report-button.png)

### Generate report from the command line

To automatically generate a report from the command line once the test finishes running, use the `K6_WEB_DASHBOARD_EXPORT` option. For example:

```shell
K6_WEB_DASHBOARD=true K6_WEB_DASHBOARD_EXPORT=html-report.html k6 run script.js
```
````

@oleiade just a suggestion here taking into account @szkiba feedback to mention `K6_WEB_DASHBOARD_EXPORT`.